language-configuration.json

```diff
@@ -1,11 +1,26 @@
 {
 	"comments": {
-		"lineComment": "///"
+		"lineComment": "///",
+        "blockComment": ["/*", "*/"]
 	},
-	"brackets": [["{", "}"], ["[", "]"], ["(", ")"]],
+	"brackets": [
+        ["{", "}"], 
+        ["[", "]"], 
+        ["(", ")"]
+    ],
 	"autoClosingPairs": [
     { "open": "{", "close": "}" },
     { "open": "[", "close": "]" },
-		{ "open": "(", "close": ")" }
-	]
+	{ "open": "(", "close": ")" },
+    { "open": "\"", "close": "\"", "notIn": ["string"]},
+    { "open": "'", "close": "'", "notIn": ["string", "comment"]},
+    { "open": "/**", "close": " */", "notIn": ["string"]}
+	 ],
+    "surroundingPairs": [
+        ["(", ")"],
+        ["[", "]"],
+        ["{", "}"],
+        ["\"", "\""],
+        ["'", "'"]
+    ]
```

Comment:
I think these changes look good, but please format the file so everything is indented correctly :)